modules/rosa-delete-objects.adoc

```diff
@@ -0,0 +1,273 @@
+
+[source,terminal]
+----
+$ rosa delete admin --cluster=mycluster
+----
+
+[id="rosa-delete-cluster_{context}"]
+== delete cluster
+
+Deletes a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa delete cluster [ID | Name] [arguments]
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster to delete.
+
+|--watch
+|Watches the cluster uninstallation logs.
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|-i, --interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+
+|-y, --yes
+|Automatically answers yes to confirm the operation.
+|===
+
+.Examples
+Delete a cluster named 'mycluster'.
+
+[source,terminal]
+----
+$ rosa delete cluster mycluster
+----
+
+Delete a cluster using the --cluster argument.
+
+[source,terminal]
+----
+$ rosa delete cluster --cluster=mycluster
+----
+
+[id="rosa-delete-idp_{context}"]
+== delete idp
+
+Deletes a specific identity provider (IDP) from a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa delete idp [ID | Name] [arguments]
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster from which the IDP will be deleted.
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|-i, --interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+
+|-y, --yes
+|Automatically answers yes to confirm the operation.
+|===
+
+.Example
+Delete an identity provider named github-1 from a cluster named 'mycluster'.
+
+[source,terminal]
+----
+$ rosa delete idp github-1 --cluster=mycluster
+----
+
+[id="rosa-delete-ingress_{context}"]
+== delete ingress
+
+Deletes a non-default application router (ingress) from a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa delete ingress [arguments]
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster from which the ingress will be deleted.
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enable debug mode.
+
+|-i, --interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+
+|-y, --yes
+|Automatically answers yes to confirm the operation.
+|===
+
+.Examples
+Delete an ingress with ID a1b2 from a cluster named 'mycluster'.
+
+[source,terminal]
+----
+$ rosa delete ingress --cluster=mycluster a1b2
+----
+
+Delete a secondary ingress using the sub-domain name.
```

Comment:
So is this deleting by passing in the subdomain name as opposed to the ID in the previous example? If so, I'd be consistent in the wording with something like:

```
 Delete an ingress with a subdomain name of `apps2` from a cluster named `mycluster`.
```

Also I believe it's subdomain, no hyphen

Reply:
Done.